02-account-kit-instantiate-usage/index.js

```diff
@@ -9,6 +9,9 @@ async function main() {
     // Retrieves data from the APIs (v1/chain/get_account)
     await account.loadData()
 
+    // Or do the above two steps in one step (instantiates object + calls load data)
+    const account: Account = await Account.load('teamgreymass', Chains.EOS)
```

Comment:
This combines the two steps above into one, adding `await` and then automatically calling `.loadData` once it's created.

It's still has the idea of an account representation idea.

- `Account.from()` will create the representation.
- `await Account.load()` will create the representation and automatically load data.